RFC Draft.md

```diff
@@ -167,37 +167,25 @@ Amplify is proposing the following changes for the `Auth` category. Similar chan
 
 User attributes inference on the `signUp` API.
 
-**Current Usage (v5)**
-
 ```Typescript
 Auth.signUp({
-  username: 'username',
-  password: '*******',
-  attributes: {
-    email: 'email@domain.com'
-  }
+  username: "username",
+  password: "*******",
+  options: {
+    userAttributes: {
+      email: "email@domain.com",
+    },
+  },
 });
 ```
 
 **Current DX (v5)**
 
 ![signup-v5](https://user-images.githubusercontent.com/70438514/222456624-b4af7349-db9d-4304-9cdc-08cf649e2d31.png)
 
-**Proposed Usage (v6)**
```

Comment:
Ditto here for the proposed v6 usage? Or is the idea v5 & v6 are the same now?

Reply:
just include usage for v5 and v6